### PR TITLE
feat : change yap to v2

### PR DESCRIPTION
**Pull request by huangsihao7**

### CODE REVIEW

File: cmd/gopcomm/post_api_article_commit.yap [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	"github.com/goplus/community/translation"
+	"github.com/qiniu/x/xlog"
+	"strconv"
+)
+
+var (
+	trans *translation.Engine
+)
+
+xLog := xlog.New("")
```
Comment (CarlJi) [inline cmd/gopcomm/post_api_article_commit.yap]: use https://github.com/qiniu/x/tree/main/log, no need to initialize

Comment (CarlJi) [reply cmd/gopcomm/post_api_article_commit.yap]: we can also organize these variables into var scope

File: cmd/gopcomm/get.yap [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	limitConst = 10
+	labelConst = "article"
+)
+
+xLog := xlog.New("")
+todo := c.TODO()
+var user *core.User
+token, err := Request.Cookie("token")
+if err == nil {
+	user, _ = community.GetUser(token.Value)
```
Comment (CarlJi) [inline cmd/gopcomm/get.yap]: handler err

File: cmd/gopcomm/get.yap [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import (
+	c "context"
```
Comment (CarlJi) [inline cmd/gopcomm/get.yap:2]: group the packages

File: cmd/gopcomm/delete_api_media_#id.yap [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+token, err := Request.Cookie("token")
+uid, err := community.ParseJwtToken(token.Value)
+if err != nil {
+	xLog.Error("token parse error")
+	json {
+		"code": 0,
+		"err":  err.Error(),
+	}
+}
+err = community.DelMedia(todo, uid, id)
```
Comment (CarlJi) [inline cmd/gopcomm/delete_api_media_#id.yap]: community.DelMedia(context.TODO(), uid, id)

File: cmd/gopcomm/get_api_video_#id.yap [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import (
+	c "context"
+	"fmt"
+	"regexp"
+)
+
+id := param("id")
+fileKey, err := community.GetMediaUrl(c.TODO(), id)
```
Comment (CarlJi) [inline cmd/gopcomm/get_api_video_#id.yap:8]: need handle err

**Review by CarlJi — APPROVED** [review]
LGTM

File: cmd/gopcomm/post_api_article_commit.yap [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+		Abstract: abstract,
+		Label:    label,
+	},
+	Content: content,
+	Trans:   trans,
+	VttId:   param("vtt_id"),
+}
+
+if trans {
+	article, _ = community.TranslateArticle(c.TODO(), article)
```
Comment (IRONICBo) [inline cmd/gopcomm/post_api_article_commit.yap]: shadow error?

**Review by IRONICBo — APPROVED** [review]
LGTM

**Pull request merged**